data/saas/dataset/gorgias_dataset.yml

```diff
@@ -0,0 +1,48 @@
+dataset:
+  - fides_key: gorgias_connector_example
+    name: Gorgias Dataset
+    description: A sample dataset representing the Gorgias connector for Fidesops
+    collections:
+      - name: customer
+        fields:
+          - name: id
+            data_categories: [user.derived.identifiable.unique_id]
+            fidesops_meta:
+              primary_key: True
```

Comment:
Nice flagging of the `primary_key` for these collections.

Reply:
Thank you